src/olympia/addons/tests/test_tasks.py

```diff
@@ -208,13 +208,15 @@ def test_update_addon_hotness():
     addon1 = addon_factory(hotness=0, status=amo.STATUS_APPROVED)
     addon2 = addon_factory(hotness=123, status=amo.STATUS_APPROVED)
     addon3 = addon_factory(hotness=123, status=amo.STATUS_AWAITING_REVIEW)
+    addon4 = addon_factory(hotness=123, status=amo.STATUS_AWAITING_REVIEW)
```

Comment:
Why are we adding a 4th add-on that is the same as the 3rd add-on?

Reply:
I meant to make it deleted but forgot... I'll fix that.